quarkus/distribution/build.gradle.kts

```diff
@@ -58,12 +58,16 @@ distributions {
         contents {
             // Copy admin distribution contents
             into("admin") {
-                from(adminDistribution)
+                from(adminDistribution) {
+                    exclude("quarkus-app-dependencies.txt")
```

Comment:
It is probably preferable to do this excluding in the server and admin modules.

Reply:
Tried to exclude it in server and admin module. Didn't find a clean solution. It may not be worth it. Let me know if you got better solutioin.

Reply:
I still think this is sub-optimal from the gradle build perspective, but I do not have a solution ATM, so I'm fine merging this change "as is" unless someone has a solution handy.